addons/fools/index.js

```diff
@@ -0,0 +1,277 @@
+export default async function ({ addon, global, console }) {
+  document.addEventListener("keydown", async (e) => {
+    if (e.key == "s") {
+      await setup();
+    }
+  });
+  async function setup() {
+    await wait(300);
+    document.body.style.overflow = "hidden";
+    // create full screen canvas
+    const canvas = document.createElement("canvas");
+    canvas.width = window.innerWidth;
```

Comment:
The page's content doesn't get wider than 942px which means no thumbnails so I think the canvas should be limited to around there, maybe with the score/lives still in the corners?
```suggestion
    canvas.width = 950;
```